### PR TITLE
optimized normalize() with inbuilt function to get 55.3% speedup

### PR DESCRIPTION
**Pull request by sidd-27**
used torch function for normalization and also wrote through tests for it

method    time (µs)    % improv.
--------------------------------
  orig      1231.08             —
 fnorm       792.92        35.59%

Accuracy check (max abs error vs. orig):
  fnorm : 5.960e-08

https://colab.research.google.com/drive/1ey7hEol_dpGYC6ap3-qnhygKylsEh7V8?usp=sharing
benchmarking script

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Did you update CHANGELOG in case of a major change?

### CODE REVIEW

File: kornia/geometry/plane.py [hunk 1]
```diff
@@ -31,7 +33,7 @@
 
 
 def normalized(v: Tensor, eps: float = 1e-6) -> Tensor:
-    return v / batched_dot_product(v, v).add(eps).sqrt()
+    return F.normalize(v, p=2, dim=-1, eps=eps)
```
Comment (shijianjian) [inline kornia/geometry/plane.py]: We would avoid using torch functions, since we have also functionalities to port the code to numpy, etc.

**Comment by sidd-27** [comment]
this also provides an optimization over the previous function, up to 20% in some cases. but the variance is too much so I assume the average improvement is about 6-7%, barely over the threshold @shijianjian 

def normalized(v: torch.Tensor, eps: float = 1e-6) -> torch.Tensor:
    norm_sq = (v * v).sum(dim=-1, keepdim=True) + eps
    return v * norm_sq.rsqrt()

pasted this in the original benchmarking code to get these results
